now.json

```diff
@@ -19,4 +19,4 @@
             "destination": "/api/open-positions"
         }
     ]
-}
```

Comment:
O tohle se na tom projektu stará automaticky Prettier, abysme to nemuseli řešit ručně. Takže teď by se stalo to, že až já uložím, tak by Prettier ty newlines zase sundal. V čem to upravuješ? Doporučuju třeba VS Code, tam by ti ten integrovaný Prettier mohl fungovat bez větších operací a nebudeš muset řešit whitespaces ani formátování.

Reply:
Pardon! Nedoinstaloval jsem zřejmě. Vrátím.